--- protobuf-3.14.0-org/src/google/protobuf/port_def.inc	2020-11-14 04:53:39.000000000 +0800
+++ protobuf-3.14.0/src/google/protobuf/port_def.inc	2021-01-08 18:42:05.869163755 +0800
@@ -563,7 +563,7 @@
 // Some constructors can't be constexpr under MSVC, but given that MSVC will not
 // do constant initialization of globals anyway we can omit `constexpr` from
 // them. These constructors are marked with PROTOBUF_MAYBE_CONSTEXPR
-#if defined(_MSC_VER)
+#if defined(_WIN32)
 #define PROTOBUF_MAYBE_CONSTEXPR
 #else
 #define PROTOBUF_MAYBE_CONSTEXPR constexpr
